test/transformers/test_mm_int8int2.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import torch
+
+from liger_kernel.transformers.experimental.mm_int8int2 import matmul, unpack_weights, pack_weights
+
+
+# input_features = size*4 when the weight matrix is unpacked
+@pytest.mark.parametrize(
+    "size",
+    [
+        2048,
+        1024,
+        512,
+    ],
+)
+@pytest.mark.parametrize(
+    "batch_size",
+    [
+        1,
+        2,
+        3,
+        8,
+        100
+    ],
+)
+@pytest.mark.parametrize(
+    "seq_len",
+    [
+        1,
+        7,
+        16,
+        2048
+    ],
+)
+@pytest.mark.parametrize(
+    "out_features",
+    [
+        1024,
+        2048,
+        4096,
+        10000,
+    ],
+)
+@pytest.mark.parametrize(
+    "atol, rtol, device",
+    [
+        (1e-2, 1e-2, "cuda"),
+    ],
+)
+def test_kernel_correctness(batch_size, seq_len, out_features, size, atol, rtol, device):
+    print(f"\nTesting kernel with size: {size}, atol: {atol}, rtol: {rtol}")
+
+    # Generate the random tensors
+    ht = torch.randint(-127, 127, (batch_size, seq_len, size * 4), device=device, dtype=torch.int8)
+    u = torch.randint(0, 255, (out_features, size), device=device, dtype=torch.uint8)
+
+    # Calculate dimensions
+    B, M, N = ht.size()
+
+    # Compute triton output
+    triton_output = matmul(ht.view(B * M, N), u.T.contiguous()).view(B, M, -1)
+
+    # Validate packing and unpacking of weights
+    assert (pack_weights(unpack_weights(u.T), 2) == u.T).all(), "Packed weights do not match original weights."
```

Comment:
imo we can separate the correctness of pack + unpack to another testing func

Reply:
Okay i will